python/tests/wrapper/test_01_synthetic_collection.py

```diff
@@ -0,0 +1,100 @@
+import uuid
+import shutil
+import tempfile
+import itertools
+import pytest
+
+import artm.wrapper
+import artm.wrapper.messages_pb2 as messages
+import artm.wrapper.constants as constants
+import helpers
+
+def test_func():
+    # Set some constants
+    num_tokens = 60
+    num_items = 100
+    pwt = 'pwt'
+    nwt = 'nwt'
+    
+    num_topics = 10
+    num_inner_iterations = 10
+    num_outer_iterations = 10
+    num_top_tokens = 4
+
+    perplexity_tol = 0.001
+    expected_perplexity_value_on_iteration = {
+        0: 55.620,
+        1: 37.397,
+        2: 28.466,
+        3: 23.285,
+        4: 20.741,
+        5: 20.529,
+        6: 20.472,
+        7: 20.453,
+        8: 20.454,
+        9: 20.455
+    }
+
+    top_tokens_tol = 0.05
+    expected_top_tokens_weight = 0.1
+
+    batches_folder = tempfile.mkdtemp()
+    try:
+        # Generate small collection
+        batch = messages.Batch()
+        batch.id = str(uuid.uuid4())
+        for token_id in xrange(num_tokens):
+            batch.token.append('token_{0}'.format(token_id))
+
+        for item_id in xrange(num_items):
+            item = batch.item.add()
+            item.id = item_id
+            field = item.field.add()
+            for token_id in xrange(num_tokens):
+                field.token_id.append(token_id)
+                background_count = ((item_id + token_id) % 5 + 1) if (token_id >= 40) else 0
+                target_topics = num_topics if (token_id < 40) and ((token_id % 10) == (item_id % 10)) else 0
+                field.token_count.append(background_count + target_topics)
+
+        # Create the instance of low-level API and helper object
+        lib = artm.wrapper.LibArtm()
+        helper = helpers.TestHelper(lib)
+
+        # Save batch on the disk
+        lib.ArtmSaveBatch(batches_folder, batch)
+
+        # Create master component and scores
+        config = messages.TopTokensScoreConfig()
+        config.num_tokens = num_top_tokens
+        scores = [('PerplexityScore', constants.ScoreConfig_Type_Perplexity, messages.PerplexityScoreConfig()),
+                  ('TopTokensScore', constants.ScoreConfig_Type_TopTokens, config)]
+        master_id = helper.create_master_component(scores=scores)
+
+        # Initialize model
+        helper.initialize_model(master_id, pwt, num_topics, source_type='batches', disk_path=batches_folder)
```

Comment:
master_id is a common parameter to many methods in helper. I suggest to make it optional (e.g. default to None), and add a setter helper.master_id = ...